web/src/integTest/resources/testSettings.properties

```diff
@@ -1,3 +0,0 @@
-maxWaitTime=180
```

Comment:
These settings should be external to the test, thus should be placed in a properties file.
Also, I might want the build to generate this file, so please revert this change and use this file instead of the constants you defined.